### PR TITLE
New Yoda Spice

### PR DESCRIPTION
**Pull request by thejasbabu**
https://duck.co/ia/view/yoda_translator

### CODE REVIEW

**Comment by daxtheduck** [comment]
@thejasbabu Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by thejasbabu** [comment]
@zekiel Hey, this is the one I was talking about. Yoda translator. Please review it. :)

**Comment by thejasbabu** [comment]
@GuiltyDolphin Please review this as well :)

**Comment by GuiltyDolphin** [comment]
@thejasbabu We should change the IA Page from `yoda_translator` to `yoda_speak`.

/cc @moollaza

File: share/spice/yoda_speak/yoda_speak.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.zci__body h3 {
+     font-size: 2.0em;
+     font-weight: 400;
+     color: #3f3f3f;
```
Comment (GuiltyDolphin) [inline share/spice/yoda_speak/yoda_speak.css]: We should assign a color via one of the classes specified in https://duckduckgo.com/styleguide.

Same with below.

File: t/YodaSpeak.t [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        caller => 'DDG::Spice::YodaSpeak'
+    ),
+    
+    'you seem to be happy to yodish' => test_spice(
+        '/js/spice/yoda_speak/you%20seem%20to%20be%20happy',
+        call_type => 'include',
+        caller => 'DDG::Spice::YodaSpeak'
+    ),
+    
+    'how does yoda say you seem to be happy' => test_spice(
```
Comment (GuiltyDolphin) [inline t/YodaSpeak.t]: We should include examples of punctuation as well as casing.

File: share/spice/yoda_speak/yoda_speak.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            },
+            meta: {
+                sourceName: "Yodish.com",
+                sourceUrl: "http://www.yodajeff.com/pages/talk/yodish.shtml" 
+            },
+           
+            templates: {
+                group: 'text',
+                options: {
+                    content: Spice.yoda_speak.yoda_speak,
```
Comment (moollaza) [inline share/spice/yoda_speak/yoda_speak.js]: Let's just stick to the text template to keep things simple. This way we can avoid using a custom template and CSS.

Comment (moollaza) [inline share/spice/yoda_speak/yoda_speak.js]: Meaning, use the standard title + subtitle approach. The title should show the answer, the subtitle should indicate what's going on: `Translation to Yoda Speak: <input>`

**Comment by zekiel** [comment]
@thejasbabu are you still working on this? Can you please take a look at moollaza's comments in line?

File: share/spice/yoda_speak/yoda_speak.handlebars [hunk 4]
```diff
@@ -0,0 +1 @@
+{{data}}
```
Comment (moollaza) [inline share/spice/yoda_speak/yoda_speak.handlebars]: @thejasbabu similar to what I mentioned in your other PR, let's skip all this custom handlebars and CSS to keep things simple and in-line with our other Instant Answers

**Comment by moollaza** [comment]
@thejasbabu I'm closing this PR for now. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**